src/views/activity.py

```diff
@@ -0,0 +1,49 @@
+
+from flask import Blueprint, render_template, request, flash, redirect, url_for, make_response, g
+from src.models.activity import Activity
+from src.models.contact import Contact
+from utility.decor import login_required
+
+
+phonebook = Contact()
+activities = Activity()
+
+activity = Blueprint('activity', __name__)
+
+@activity.route('/<int:contact_id>', methods=["GET"])
```

Comment:
This route should be `GET /contacts/:id` and needs to show both contact info and activities. Should be moved to the contacts view.

Reply:
huuum, are you sure?! Just to confirm and be more clear than codes; this function shows all activities associated with a contact_id.